data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/AbstractLogstashPluginAttributesMapperTest.java

```diff
@@ -6,27 +6,31 @@
 package org.opensearch.dataprepper.logstash.mapping;
```

Comment:
Please add a test case that verifies that the default setting is not used when it is provided by the Logstash configuration.